REVIEW.md

```diff
@@ -0,0 +1,209 @@
+    - Could the code easily be rewritten to run much more efficiently for relevant settings?
+    - Is the code backwards compatible with previous versions?
+      (Or is a deprecation cycle necessary?)
+    - Is there feedback that would be helpful for solving future issues surrounding this change?
+      Are there non-urgent concerns that the team should be made aware of?
+    - If a bug is fixed, does the PR include a non-regression test?
+    - Are there images that should be added, resized or optimized?
+
+- [ ] User-facing changes
+    - Does the user interface or other visual elements match existing patterns in Spyder?
+      If it does not, is there a reason?
+
+Review that addresses these questions on a PR will be prefixed with `[optional]` so that the PR author(s) are clear that this feedback is not a blocker.
+Reviews may be tagged per comment or through the primary comment of a GitHub review.
+Addressing this feedback is considered above-and-beyond, and may be more relevant on some PRs than others.
+PR authors may address these reviews, but are not required to do so in order for the PR to be eligible for merging.
+
+#### Discussion items
+
+I am not sure what people want for the discussion items tag, so this is a placeholder to make space for collaboration.
+
+### For documentation/website-focused review
+
+#### Requested items
+
+The following are areas the community expects to be covered in the review of a documentation-focused PR.
+In general, reviewers need to consider whether the content is a good fit, accurate and generally clear to readers, and builds and renders with no simmediate user-visible issues.
+
+Review that addresses these questions on a PR will be prefixed with `[requested]` so that the PR author(s) are clear that they are blockers.
+Reviews may be tagged per comment or through the primary comment of a GitHub review.
+PR authors are responsible for addressing requested reviews for the PR to be eligible for merging.
+
+**Motivation**
+
+- [ ] Does the change improve the documentation?
+- [ ] Would it benefit a meaningful number of users to have this as a part of the documentation?
+- [ ] Is it appropriate for the documentation and the section it is placed in?
+
+**Content**
+
+- [ ] Is the information accurate?
+- [ ] Is the change sufficiently complete, such that it won’t confuse users if it were to appear in the documentation as-is?
+- [ ] Is it clear what the text and images are trying to communicate?
+
+**Images/GIFs**
+
+- [ ] Are screenshots/GIFs reasonably legible and appropriate for the topic?
+- [ ] Are all images/GIFs scaled and optimized?
+      If needed, reviewers should offer to help with this.
+
+**Technical**
+
+- [ ] Do the tests pass in the continuous integration build?
+      If appropriate, help the contributor understand why tests failed.
+- [ ] Does any added/changed content render correctly, without user-visible issues?
+- [ ] Is the content free from typos and unambiguous grammar errors?
+
```

Comment:
```suggestion

In addition to requested changes, this category also includes questions that the reviewer has for the contributor that may relate to one of these areas, or otherwise need to be answered before the PR can be merged.

```